graylog2-web-interface/src/components/search/SearchResult.jsx

```diff
@@ -195,23 +215,14 @@ const SearchResult = React.createClass({
           />
         </Col>
         <Col md={9} sm={12} id="main-content-sidebar">
-          <FieldStatistics ref="fieldStatisticsComponent"
```

Comment:
Can't add it to the import sentences (as they weren't modified by the changes in the PR), but we don't need to import `FieldGraphs`, `FieldQuickValues`, and `FieldStatistics` directly here any more.